3.Tests/UI/StepDefinitions/Item/AddAboveStepDefinitions.cs

```diff
@@ -15,49 +15,41 @@ public class AddAboveItemStepDefinitions : CommonSteps
     {
         private readonly HomePage _homePage;
         private readonly ScenarioContext _scenarioContext;
-        private string _itemName = "";
+        private readonly string _itemName = "";
```

Comment:
If this field is not used should be removed please.

Reply:
Done, thank you